.github/functional_tests/lake_test_args/action.yml

```
@@ -0,0 +1,74 @@
+name: 'Lake Test Args'
+description: 'Run `lean-action` on with `test-args` input'
+inputs:
+  toolchain:
+    description: 'Toolchain to use for the test'
+    required: true
+runs: 
+  using: 'composite'
+  steps:
+    # TODO: once `lean-action` supports just setup, use it here
+    - name: install elan
+      run: |
+        set -o pipefail
+        curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh -s -- -y --default-toolchain ${{ inputs.toolchain }}
+        echo "$HOME/.elan/bin" >> "$GITHUB_PATH"
+      shell: bash
+
+    - name: create lake package with `lake init ${{ inputs.lake-init-arguments }}`
+      run: |
+        lake init buildargs .lean
+        lake update
+      shell: bash
+
+    - name: "run `lean-action` with test-args"
```

I believe this step is failing because there isn't a test runner/driver. You should be able to use the same "create successful dummy test" step from the `lake_test_success` test: https://github.com/leanprover/lean-action/blob/main/.github/functional_tests/lake_test_success/action.yml#L24

```suggestion
    - name: create successful dummy test
      run: |
        {
          echo "@[test_runner]"
          echo "script dummy_test do"
          echo "  println! \"Running fake tests...\""
          echo "  println! \"Fake tests passed!\""
          echo "  return 0" 
        } >> lakefile.lean
      shell: bash
     
    - name: "run `lean-action` with test-args"
```